test/Spaces/sphere.jl

```diff
@@ -52,6 +52,67 @@ using Test
     end
 end
 
+@testset "Bubble correction Nq robustness" begin
+
+    for FT in (Float64, Float32)
+        no_bubble_rtols = (
+            FT(0.64),
+            FT(0.19),
+            FT(0.027),
+            FT(0.0049),
+            FT(0.0008),
+            FT(0.00014),
+            FT(0.00028),
```

Comment:
Kind of odd that this one is much larger. We have roughly 2nd order convergence, except for this one:
```julia
FT = Float64

rtols = [FT(0.64),
                  FT(0.19),
                  FT(0.027),
                  FT(0.0049),
                  FT(0.0008),
                  FT(0.00014),
                  FT(0.00028),
                  FT(3.97e-6),
                  FT(6.77e-7),]

rtols ./ (0.8 .* (0.5^2) .^ (0:8))

```

Reply:
Indeed it wasn't with `Float64`. I had to adjust it for `Float32`

Reply:
what are the corresponding numbers for `Float64`?

Reply:
It would be 
```julia
FT = Float64
rtols = [FT(0.6324474030521388),
         FT(0.1897709793144625),
         FT(0.026553449932969358),
         FT(0.0048235168713260555),
         FT(0.0007970936271045722),
         FT(0.00013703279345387713),
         FT(2.322509854565059e-5),
         FT(3.966069653787797e-6),
         FT(6.761430087433387e-7),]
```

Reply:
But it's possible that I had a typo there. Let me double check :eyes:

Reply:
Oh, yeah, definitely a typo. Sorry about that. I am going to fix it shortly. 

Even with `Float32` it would be:

```julia
FT = Float32
rtols = [FT(0.6324476f0),
         FT(0.18977092f0),
         FT(0.026553009f0),
         FT(0.004823429f0),
         FT(0.00079736137f0),
         FT(0.00013680616f0),
         FT(2.3947823f-5),
         FT(3.6427675f-6),
         FT(4.0475194f-7),]
```